src/assets/content.tsx

```diff
@@ -110,25 +200,25 @@ const content = {
       {
         icon: 'water',
         title: 'Bäume bewässern',
-        content:
+        description:
           'Informiere Dich auf unserer Plattform, ob die Bäume in deiner Straße Wasser benötigen. Wenn ja, schnapp Dir eine Gießkanne, einen Eimer oder einen Schlauch und leg los. Danach trägst du die Bewässerung hier ein.',
       },
       {
         icon: 'subscribe',
         title: 'Bäume abonnieren',
```

Comment:
In issue #176 and in the Slack channel we talked about renaming that to "adoptieren". Maybe we change this here as well and do the renaming together with this PR